### ISSUE TITLE
Progress bar in CSS instead of JS

### ISSUE THREAD

**Issue by Chouchen** [wallabag/wallabag#3194]
Something like this would be nice : https://css-tricks.com/fun-viewport-units/

**Comment by j0k3r** [comment]
To be more explicit instead of that one line sentence, I guess you are talking to [that example](https://css-tricks.com/fun-viewport-units/#article-header-id-5) specifically?

**Comment by Chouchen** [comment]
Yes, that example is the goal of the article. And that's just an example.
I find the JS version on mobile a bit clanky, which lead to that suggestion.

I hoped someone would explain more which explains the brevity of my original post. (I'm not a front dev)
